### PR TITLE
add liquidprompt HEAD

### PR DESCRIPTION
**Pull request by augmentedfourth**
Liquidprompt hasn't cut a release tag in a while, but the develop branch is actually working quite well (I've been testing it for a while).

### CODE REVIEW

**Comment by augmentedfourth** [comment]
I don't know what's going on here. My first commit had 2 errors: duplicating a core formula name, and no 'test do' block. I added a 'test do' block, and that error went away. After that, I changed the formula name, and then it wasn't duplicating anymore but it complained that it had no tests.

I'm wondering if there's a problem with the PR because I originally sent it in with one filename but now it's got a different filename. If I don't hear anything in this ticket for a while, I might close the PR and try again without all the intermediate steps.

File: liquidprompt-develop.rb [hunk 1]
```diff
@@ -0,0 +1,31 @@
+class Liquidprompt < Formula
```
Comment (dunn) [inline liquidprompt-develop.rb]: Yeah, the classname has to match the filename; `LiquidpromptDevelop` should work.

Comment (augmentedfourth) [inline liquidprompt-develop.rb]: Updated & tests seems to pass; thanks!

**Comment by dunn** [comment]
Why not just add 
```ruby
head "https://github.com/nojhan/liquidprompt.git", :branch => "develop"
```
to the formula in core, though?

**Comment by augmentedfourth** [comment]
Probably a better idea. I didn't even know that was an option.

**Comment by dunn** [comment]
We also have a `devel` option, if there are tagged releases on that branch (see https://github.com/Homebrew/homebrew/blob/master/Library/Formula/asio.rb#L14-L17)!

**Comment by augmentedfourth** [comment]
Submitted to core in https://github.com/Homebrew/homebrew/pull/45647

**Pull request closed without merging**